integration/cmd_type_test.go

```diff
@@ -158,24 +158,24 @@ func TestCreateIndexTypeWithStargate(t *testing.T) {
 		)),
 	))
 
-	env.Must(env.Exec("create a type",
+	env.Must(env.Exec("create a map",
 		step.NewSteps(step.New(
-			step.Exec("starport", "type", "user", "email", "--module", "example"),
+			step.Exec("starport", "s", "map", "user", "email"),
```

Comment:
```suggestion
			step.Exec("starport", "s", "list", "user", "email"),
```

This test actually checks you can scaffold a map when a list with the same name already exist (shouldn't be related to the current error in CI though)